conda/build.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
```

Comment:
It's never directly invoked, only implicitly by `conda build`, so some comments on what line does what should suffice.